README.md

```diff
@@ -471,39 +471,40 @@ const pk: BBSPublicKey;
 // The signature
 const sig: BBSSignature = ...;
 
-// Prover prepares the attributes he wants to disclose, i.e. attribute index 2 and 4 (indexing is 0-based), and the ones he wants to hide. 
+// Prover prepares the attributes he wants to disclose, 
+// i.e. attribute index 2 and 4 (indexing is 0-based), and the ones he wants to hide. 
 const revealedMsgIndices: Set<number> = new Set();
 revealedMsgIndices.add(2);
-revealedMsgIndices.add(4);
+revealedMsgIndices.add(3);
 
 // revealedMsgs are the attributes disclosed to the verifier
 const revealedMsgs: Map<number, Uint8Array> = new Map();
 revealedMsgs.set(2, messages[2]);
+revealedMsgs.set(3, messages[3]);
```

Comment:
Thanks. I missed it. But should be be 4.

Reply:
You're right. it's fixed now.

Reply:
Thanks.